gammapy/cube/core.py

```diff
@@ -218,6 +218,22 @@ def empty(cls, emin=0.5, emax=100, enumbins=10, eunit='TeV', mode='edges', **kwa
         kwargs : dict
             Keyword arguments passed to `~gammapy.image.SkyImage.empty` to create
             the spatial part of the cube.
+        
+        Examples
+        --------
+        Create an empty sky cube:
+
+        .. code::
+
+            from gammapy.cube import SkyCube
+            cube = SkyCube.empty(nxpix=11, nypix=7, enumbins=3, mode='center',
+                            emin=1, emax=100, eunit='TeV')
```

Comment:
Put `enumbins` on the second line with the other energy-related arguments?

Also: In docstrings I think I'd be in favour of just doing the double-colon instead of `.. code::`.
The rendered HTML is the same, and the docstring is a bit simpler / shorter (it's what Numpy, Scipy, Astropy does in most (all?) examples sections in docstrings.
```
Create an empty sky cube::

    <code here>
```